geode-core/src/main/java/org/apache/geode/management/cli/GfshCommand.java

```diff
@@ -87,6 +88,10 @@ public Cache getCache() {
     return locator == null ? null : (T) locator.getConfigurationPersistenceService();
   }
 
+  public LocatorClusterManagementService getClusterConfigurationService() {
+    InternalLocator locator = InternalLocator.getLocator();
```

Comment:
For `GfshCommand`, we haven't gone through the effort of removing these static lookups yet. In fact, `GfshCommand` came into existence because we want to hide these static lookups in one place. In here, not only `locator`, `gfsh`, and `managementService` are also accessed through static methods.